Diona Species Design Document (#606)
space-wizards/docs: src/en/space-station-14/character-species/proposals/diona.md

# Diona Species

| Designers | Implemented   | GitHub Links                                                                      |
|---|---------------|-----------------------------------------------------------------------------------|
| Tuerk | :x: No | TBD |

## Overview

> This proposal covers the Diona species as a playable roundstart species. Dionae are a type of of humanoid/plantoid alien species. Though they have many different forms and personalities, these "components" remain the same: planty appearance, modular, omnivores and high powered radiothropy.

Reviewer: ```suggestion
This proposal covers the Diona species as a playable roundstart species. Dionae are a type of humanoid/plantoid alien species. Though they have many different forms and personalities, these "components" remain the same: planty appearance, modular, omnivores, and high-powered radiothropy.
```

This document is meant to set guidelines for future PRs to make Diona stand out in their own way from other species available in WizDen, through aesthetics and unique gameplay found nowhere else.

## Design

Playing a Diona should feel much different from any other species because of their inherent traits, the degree of which should be about equal to playing a Vox (who are required to breathe nitrogen at all times, but are otherwise unhindered from doing any job on the station). This is done by emphasising their unusual biology as a composite being.

A large variety of cosmetic options should also allow players to play as many different plantoid/fungoid species they can come up with.

### Core Visual Elements

Diona appear to be made out of plant-like matter, though they are made up of insect/slug-like Nymphs, represented by the multitude of eyes visible throughout their body, connected through viney biomass secretions. All working Dionae are required to have a humanoid (cyclops) form before hiring.

> They communicate in a singsong manner, with noises created through their own biomass that remind of natural sounds like rustling leaves, rythmic cracking, branches rubbing against each other, insect chirping, to less natural sounds like plucked strings or playing a flute. Diona are still able to mimic facial features or appendages like fingers or tails to communicate with other races, by designating a nymph to create those features.

Reviewer: ```suggestion
They communicate in a singsong manner, with noises created through their own biomass that remind one of natural sounds like rustling leaves, rhythmic cracking, branches rubbing against each other, insect chirping, to less natural sounds like plucked strings or playing a flute. Dionae can still mimic facial features or appendages, such as fingers or tails, to communicate with other races by designating a nymph to create those features.
```

### Naming Scheme

> Diona always give themselves names. These names are often long and intricate, representing the journey they have gone on, experiences had, important emotions, actions or ideals, important locations, interactions had or daily routines. So to say their name should reflect their own identity, which also means it can change as they live.

Reviewer: ```suggestion
Diona always give themselves names. These names are often long and intricate, representing the journey they have gone on, the experiences they have had, important emotions, actions or ideals, important locations, interactions, or daily routines. So, to say, their name should reflect their identity, which also means it can change as they live.
```

If you would prefer this part to be improved somewhat.

> For practical reasons, Diona shorten their names to make them more managable, sharing their entire name often only to other Dionae or close friends.

Reviewer: ```suggestion
For practical reasons, Diona shorten their names to make them more manageable, sharing their full names only with other Dionae or close friends.
```

Examples, some of which are taken from Aurorastation, others I've found fitting:

- Reveletation of Self (Pensive Exultation Regarding Astonishing Revelation of Self)
- Sanguis (Rationing Knowledge of Embodied Sanguis)
- Ecclesiarch Devoted Emissary of Divine Benediction (Does not use a shortened name!)
- Explorers (Explorers Of An Everchanging Conundrum)
- Purple (Purple Blades Vanishing Into The Horizon)
- Leaf (Crashes Through Heaven As A Leaf Through Trees)
- Honk (Honks All Security Through The Voids And Stars)

> Note! Nicknames are also acceptable! If your Diona grew up with other races, names adopted from other species are also possible! Go Wild!

### Species Features

> Their unique biology gives them several advantages as well as severe downsides. Laying it all out here is an effort, and I fully expect most of these features will need to be rolled out piece-meal.

Reviewer: ```suggestion
Their unique biology gives them several advantages and severe downsides. Laying it all out here is an effort, and I fully expect most of these features will need to be rolled out piecemeal.
```

> In Aurorastation, this species uses a Biomass resource mechanic to regrow limbs or create structures not unlike Xenomorphs, which I have elected to leave out.
> > The former, to create more interaction with medical and comform with design Rule 3b: Job Impact. It will be interesting gameplay and RP for medical players to operate on a diona's individual components.
> >
> > The latter is left out to comform with Rule 1: A Distinct Identity; further differentiate Diona from Arachnids and their ability to build structures out of web.
> > Personally, I would enjoy Arachnids being able to make nests, while Diona create a sort of greenhouse/forest in a station, but this would require the biomass resource, and I am debating the technical feasability on it.

#### Aesthetics

> Having access to biomass secretions, the Diona is able to express themselves in many different ways by selecting from many different markings like aesthetic flowers on their arms or head and different growths on their head, for example a mushroom cap. Through the use of multiple of these markings, players can create a character that reminds of any plant or fungi they wish, or go any other direction.

Reviewer: ```suggestion
Having access to biomass secretions, the Diona can express themselves in many different ways by selecting from a range of markings, such as aesthetic flowers on their arms or head, and different growths on their head, for example, a mushroom cap. Through the use of multiple of these markings, players can create a character that reminds them of any plant or fungi they wish, or go any other direction.
```

#### Diona Nymphs

> *Henceforth when I refer to Diona, the humanoid player characters are meant, whereas Diona Nymphs are called so explicitly.*

Reviewer: ```suggestion
*Henceforth, when I refer to Diona, the humanoid player characters are meant, whereas Diona Nymphs are called so explicitly.*
```

Diona player characters are made up of a multitude of Nymph Diona. These are dog sized slugs, individually only as smart and inquisite as a corgi. Connected through biomass they make up a sentient humanoid Diona, smarter than the whole. As the personality of a Diona is made up of the individual Nymphs they make up, it's in the best interest of the Diona to remain whole.

Diona Nymphs are NPCs created by one of the following ways:

- a Diona creating a Seeding Nymph in a hydroponics bay (with some drawback and long cooldown), basically sacrificing a mature Nymph to create a very fragile plant that can be harvested a couple times which yields a Nymph as fruit.
- "Maints Nymphs", mapped in maps with overgrown areas or greenhouses.
- a Diona gibbing themselves either while dead / at will (expanded on further down)

The donation of a Nymph to help a Diona or delimbed Crew in need is vanishingly rare, but in cases of great trust and need not unheard of.

> > For example, if a crew is missing an arm, a diona Nymph can be attached on the stump which will then work like a normal arm. This Nymph will be in a dormant state, connecting with the nervous system of the host to work like a prosthetic. They can be just as easily detatched however with no harm to host or Nymph.

Reviewer: ```suggestion
> For example, if a crew is missing an arm, a diona Nymph can be attached to the stump, which will then work like a normal arm. This Nymph will be in a dormant state, connecting to the host's nervous system to function as a prosthetic. They can be just as easily detached, however, with no harm to the host or the Nymph.
```

> 
> *Surgery and delimbing of course implies the implementation of the new medical system beforehand.*

Diona value their inner community most, with the loss of one Nymph being a tragedy.´To neglect or leave a Nymph without guidance is a taboo for Diona. 

The intent behind the emphasis of Diona Nymphs is to make Diona a stand-out species besides just turning to dust when exposed to fire and looking like a walking plant, but also being an unique, unobtrusive feature that enriches each round for the player and the crew. *Currently Diona Nymphs do exist, but provide nothing but evidence that a Diona had been gibbed some time ago.*

#### Rooting

> Diona do not possess high-powered muscles like normal species, but they do possess biomass which they can manipulate up to a degree, in this case holding onto the floor with a multitude of vines, or roots. This ability lets them move like usual in zero-G scenarios like in case of an emergency or when working/sunbathing in space, making them excellent engineers or asteroid miners, however they are slowed while the ability is active and passively ingest whatever liquid is on the floor through osmosis. Usually you do not want things like glue or welding fuel in your system.

Reviewer: ```suggestion
Diona do not possess high-powered muscles like normal species, but they do possess biomass, which they can manipulate up to a degree, in this case, holding onto the floor with a multitude of vines or roots. This ability lets them move as usual in zero-G, such as during an emergency or while working/sunbathing in space, making them excellent engineers or asteroid miners. They are slowed while the ability is active and passively ingest whatever liquid is on the floor through osmosis. Usually, you do not want things like glue or welding fuel in your system.
```

Most crew do not want to see what happens to a Diona that has accidentally consumed blood of another sentient, as it can have any number of sideeffects, such as but not limited to *severe trembling, incoherent speech, narcolepsy, violent outbursts, mental decay, and/or spontanious death or gibbing.*

As a side effect of their curious form of mobility, this species is not hindered by any form of kudzu.

> I uh... mostly just put this here because it's already a thing, doesn't come up often, but would be weird if lost from a rework.

#### Terrestrial

Even with the existance of voidic Dionae, most social Dionae trace their lineage to a terrestrial genetic baseline, thus lacking most forms of space tolerance or extreme temperature protection.

> Aurorastation allows Dionae to survive in space and high pressure as long as they have an oxygen supply, but to comform with the design rule 8 "Circumvention of Intentional Game Design", I have elected to disregard this aspect.

#### Radiothropic

> There is a misconception that Diona are photosynthetic. In truth, their cells are able to generate energy from pure radiation to such a high degree, that visible radiation is sufficient for continued functioning. Without a source of light or radiation, the Diona will take damage and eventually die.

Reviewer: ```suggestion
There is a misconception that Diona are photosynthetic. In truth, their cells can generate energy from pure radiation to such an extent that visible radiation is sufficient for continued functioning. Without a source of light or radiation, the Diona will take damage and eventually die.
```

#### Conditional Omnivores

Dionae diet is about maintaining a variety of elements, minerals and calory intake rather than taste. Spent bullet casings and uranium ore are as much a part of their diet as is a salad or water.

To ingest food, they must first encase the item in question in a food vacuole, then ingest it. Uncooked meat, like blood, is harmful to the physioligy and psychology of a Diona. Meat if cooked well done is digestable.

Dionae Nymphs can consume foods directly through a sharp, extendable proboscis, while Dionae can do the same for smaller foodstuffs.

> I am unsure how feasable this is technically. It would *just* require checking the tags of the food item so that if the Meat tag is present, Cooked must also be present or else the blood-related negative effects trigger, but this could force one to check every single "cooked" foodstuffs and making sure they are tagged correctly.
>

> > Still, if this feature is deemed unfeasable, they could just be made averse to eating meat in general, though I would still want them to be able to eat inorganic material.

Reviewer: ```suggestion
> I am unsure how technically feasible this is. It would *just* require checking the tags of the food item so that if the Meat tag is present, Cooked must also be present, or else the blood-related negative effects trigger. But this could force one to check every single "cooked" food item and make sure it is tagged correctly.
>
> Still, if this feature is deemed unfeasible, they could just be made averse to eating meat in general, though I would still want them to be able to eat inorganic material.
```

"Foodstuff?"

Author: Yea, german-mind hadn't seen an issue with it x.x

Diona Nymphs are NPCs and follow the last Diona that petted them, as they rely on larger Diona forms for guidance and protection. They are created in one of the following ways:

- A Diona creating a Seeding Nymph in a hydroponics tray or similar (with an interaction, some drawback and long cooldown), basically sacrificing a mature Nymph to create a very fragile plant that can be harvested a couple of times, which yields a Nymph as fruit.
- "Maints Nymphs", mapped on stations with overgrown areas or greenhouses.
- a Diona gibbing themselves either while dead (expanded on further down in 'Unique Gibbing Behavior')

The donation of a Nymph to help a Diona or delimbed Crew in need is vanishingly rare, but in cases of great trust and need not unheard of.

> For example, if a crew is missing an arm, a diona Nymph can be attached to the stump, which will then work like a normal arm. This Nymph will be in a dormant state, connecting to the host's nervous system to function as a prosthetic. They can be just as easily detached, however, with no harm to the host or the Nymph.
> 

> > *Surgery and de-limbing, of course, imply the implementation of the new medical system beforehand.*

Author: @sowelipililimute I was imagining that a Diona Nymph limb would be the best prosthetic one could get right behind a high-quality cybernetic, but has the major downside of straight up falling off dead when it takes too much damage.

Regarding how easily prosthetics can be acquired and what negative effects it would incur on a character, could this even work with the new medical system that's being worked on?
